src/components/ServiceCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardFooter
+} from "@baltimorecounty/dotgov-components";
+
+let starImg = {
+  fontSize: ".2em",
+  float: "right",
+  marginTop: "0px",
+  width: "10px"
+};
+
+const ServiceCard = ({ name, department, url, icon, rank }) => (
+  <div className="d-flex col-lg-3 col-md-6 col-sm-6">
+    <Card>
+      <a href={url}>
+        <CardContent>
+          <div className="dg_icon-container">
+            <i className={icon} aria-hidden="true"></i>
+            <i
+              className={rank ? "fas fa-star" : ""}
+              style={starImg}
+              aria-hidden="true"
+            />
+          </div>
+          <h3>{name}</h3>
+        </CardContent>
+        <CardFooter>
```

Comment:
I'm not sure if this is a component issue or not, but the department does not match the mockup.

![image](https://user-images.githubusercontent.com/1933400/72371481-79a35200-36d2-11ea-950b-9ee559e0a2c8.png)

sam is going to get us this information.